FLT/HaarMeasure/DistribHaarChar/Padic.lean

```diff
@@ -0,0 +1,106 @@
+/-
+Copyright (c) 2024 Yaël Dillies, Javier Lopez-Contreras. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Javier Lopez-Contreras
+-/
+import FLT.Mathlib.MeasureTheory.Group.Action
+import FLT.HaarMeasure.DistribHaarChar.Basic
+import FLT.HaarMeasure.MeasurableSpacePadics
+
+/-!
+# The distributive Haar characters of the p-adics
+
+This file computes `distribHaarChar` in the case of the actions of `ℤ_[p]ˣ` on `ℤ_[p]` and of
+`ℚ_[p]ˣ` on `ℚ_[p]`.
+
+This lets us know what `volume (x • s)` is in terms of `‖x‖` and `volume s`, when `x` is a
+p-adic/p-adic integer and `s` is a set of p-adics/p-adic integers.
+
+## Main declarations
+
+* `distribHaarChar_padic`: `distribHaarChar ℚ_[p]` is the usual p-adic norm on `ℚ_[p]`.
+* `distribHaarChar_padicInt`: `distribHaarChar ℤ_[p]` is the usual p-adic norm on `ℤ_[p]`.
```

Comment:
`distribHaarChar ℤ_[p]` is a character on Z_p^* and is the constant function 1. It's not even defined on all of Z_p. This docstring is quite misleading I think?